scopes/lanes/ui/readme/lane-readme.module.scss

```diff
@@ -0,0 +1,26 @@
+.readmeContainer {
+  padding-bottom: 50px;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+.laneId {
+  padding: 50px 0px 0px 120px;
+}
+.emptyLane {
+  height: auto;
+  padding-top: 8px;
+  padding-bottom: 16px;
+}
+.readmeComponentCard {
+  width: 300px;
```

Comment:
this will not resize in smaller screens and can cause things to overflow. maybe use `max-width`?